docs/apis/avalanchego/apis/subnet-evm.md

```diff
@@ -0,0 +1,186 @@
+---
+sidebar_position: 15
+---
+
+# Subnet-EVM API
+
+[Subnet-EVM](https://github.com/ava-labs/subnet-evm) APIs are identical to [Coreth](https://github.com/ava-labs/coreth) APIs, except Avalanche Specific APIs starting with `avax`. Subnet-EVM also supports standart Ethereum APIs as well. For more information about Coreth APIs see [here](api/../c-chain.md).
+
+Subnet-EVM has some additional APIs that are not available in Coreth.
+
+## eth_feeConfig
+
+Subnet-EVM comes with an API request for getting fee config at a specific block. You can use this API to check your activated fee config.
+
+**Signature**
+
+```sh
+eth_feeConfig([blk BlkNrOrHash]) -> {feeConfig: json}
+```
+
+- `blk` is the block number or hash at which to retrieve the fee config. Defaults to the latest block if omitted.
+
+**Example Call**
+
+```sh
+curl -X POST --data '{
+    "jsonrpc": "2.0",
+    "method": "eth_feeConfig",
+    "params": [
+        "latest"
+    ],
+    "id": 1
+}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/bc/2ebCneCbwthjQ1rYT41nhd7M76Hc6YmosMAQrTFhBq8qeqh6tt/rpc
+```
+
+**Example Response**
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": {
+    "feeConfig": {
+      "gasLimit": 8000000,
+      "targetBlockRate": 2,
+      "minBaseFee": 33000000000,
+      "targetGas": 15000000,
+      "baseFeeChangeDenominator": 36,
+      "minBlockGasCost": 0,
+      "maxBlockGasCost": 1000000,
+      "blockGasCostStep": 200000
+    },
+    "lastChangedAt": 0
+  }
+}
+```
+
+## eth_getChainConfig
+
+`eth_getChainConfig` returns chain config with parsed upgrade bytes. This API is enabled by default with `internal-blockchain` namespace.
+
```

Comment:
```suggestion
`eth_getChainConfig` returns the Chain Config of the blockchain. This API is enabled by default with `internal-blockchain` namespace.

This API exists on the C-Chain as well, but in addition to the normal Chain Config returned by the C-Chain `eth_getChainConfig` on subnet-evm additionally returns the upgrade config, which specifies network upgrades activated after the genesis.
```

Suggested this to make it more clear what the difference is between this API on the C-Chain and subnet-evm.